packages/opentelemetry/src/utils/parseSpanDescription.ts

```diff
@@ -170,11 +189,14 @@ export function descriptionForHttpMethod(
   const origin = attributes[SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN] || 'manual';
   const isManualSpan = !`${origin}`.startsWith('auto');
 
-  const useInferredDescription = isClientOrServerKind || !isManualSpan;
+  // If users (or in very rare occasions we) set the source to custom, we don't overwrite it
+  const alreadyHasCustomSource = attributes[SEMANTIC_ATTRIBUTE_SENTRY_SOURCE] === 'custom';
+
+  const useInferredDescription = !alreadyHasCustomSource && (isClientOrServerKind || !isManualSpan);
```

Comment:
just to be clear - we always set this to custom for `startSpan()` calls by default, right? So that means span names will never be inferred for spans started with this, is this right? IMHO that's OK, just wanted to clarify :)

Reply:
yes, that's correct.

Reply:
This case is also covered by a test: https://github.com/getsentry/sentry-javascript/blob/2ed0509850dd2615b4be030f8e4a9676d413fc7e/packages/opentelemetry/test/utils/parseSpanDescription.test.ts#L438-L458 and I added another integration test for it.

Didn't find problematic code paths in our code